docs/README.md

```diff
@@ -1,6 +1,5 @@
 # 🍻 Tavern Keeper
-[![Tavern Keeper](https://img.shields.io/badge/Tavern_Keeper-Visit_Website-purple?style=for-the-badge)](INSERT-DEPLOY-LINK)
-<!-- Deployed Site TBD -->
+[![Tavern Keeper](https://img.shields.io/badge/Tavern_Keeper-Visit_Website-purple?style=for-the-badge)](https://tavern-keeper-fe.vercel.app/)
```

Comment:
Thanks for getting the front-end in, there should be a spot under Context where the backend can be linked as well (the repo is already linked there, just adding the deployed link as well may be beneficial).

Reply:
Sorry about the misunderstanding! I'll update those now.